molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/mote/projects/{{cookiecutter.app_name}}/website/src/patterns/basics/article-teaser-meta/element.html

```diff
@@ -0,0 +1,14 @@
+{% raw %}
+{% load mote_tags %}
+
+<div class="teaser-meta-list">
+  <div class="teaser-meta-list__likes-counter">
+    63
+    <span class="teaser-meta-list__liked-label">likes</span>
+  </div>
+  <div class="teaser-meta-list__comment-counter">
+    12
+    <span class="teaser-meta-list__comment-label">comments</span>
+  </div>
+</div>
```

Comment:
lets remove the comment counter from here, we need to add this in the molo.commenting